var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -68,10 +68,18 @@ def on_off(varstr):
             return 'ON' if varstr in self.spec else 'OFF'
 
         cmake_args = std_cmake_args[:]
+
+        fortran_flag = self.compiler.pic_flag
+        if spec.satisfies('%clang') and (sys.platform == 'darwin'):
+            libgfortran = os.path.dirname(os.popen(
```

Comment:
According to the [documentation](https://docs.python.org/2/library/os.html#os.popen), `os.popen` has been deprecated since Python 2.6. It would be better to use Spack's `Executable` class. Something like:
```python
mpif77 = Executable(self.spec['mpi'].mpif77)
libgfortran = LibraryList(mpif77('--print-file-name', 'libgfortran.a', output=str))
fortran_flag += ' ' + libgfortran.ld_flags
```